src/System.Windows.Forms/src/System/Windows/Forms/Controls/ComboBox/ComboBoxRenderer.cs

```diff
@@ -14,7 +14,11 @@ public static class ComboBoxRenderer
     // Make this per-thread, so that different threads can safely use these methods.
     [ThreadStatic]
     private static VisualStyleRenderer? t_visualStyleRenderer;
-    private static readonly VisualStyleElement s_comboBoxElement = VisualStyleElement.ComboBox.DropDownButton.Normal;
+#pragma warning disable WFO5001 // Type is for evaluation purposes only and is subject to change or removal in future updates. Suppress this diagnostic to proceed.
+    private static readonly VisualStyleElement s_comboBoxElement = Application.IsDarkModeEnabled ?
+        VisualStyleElement.CreateElement("DarkMode_CFD::COMBOBOX", 1, 1)
+        : VisualStyleElement.ComboBox.DropDownButton.Normal;
```

Comment:
The agreed styling in the repo AFAIK is as follows:
```suggestion
    private static readonly VisualStyleElement s_comboBoxElement = Application.IsDarkModeEnabled
        ? VisualStyleElement.CreateElement("DarkMode_CFD::COMBOBOX", 1, 1)
        : VisualStyleElement.ComboBox.DropDownButton.Normal;
```